tests/test_builder.py

```diff
@@ -806,3 +806,29 @@ def test_snippet_lics_info_order(self):
         self.builder.set_snippet_lics_info(
             self.document, License.from_identifier("Apache-2.0")
         )
+
+    def test_snippet_byte_range(self):
+        self.builder.create_snippet(self.document, "SPDXRef-Snippet")
+        self.builder.set_snippet_byte_range(self.document, "310:420")
+
+    @testing_utils.raises(builders.OrderError)
+    def test_snippet_byte_range_order(self):
+        self.builder.set_snippet_byte_range(self.document, "310:420")
+
+    @testing_utils.raises(builders.SPDXValueError)
+    def test_snippet_byte_range_order(self):
```

Comment:
Duplicate test name. This even causes the test not to be executed when you run the whole suite.
Also applies to line range further down

Reply:
fixed